lib/class-wp-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Patterns_Controller class
+ *
+ * @subpackage REST_API
+ * @package    WordPress
+ */
+
+/**
+ * Core class used to access block patterns via the REST API.
+ *
+ * @see   WP_REST_Controller
+ */
+class WP_REST_Block_Patterns_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wp/v2';
```

Comment:
Testing REST API endpoints, use the __experimental namespace.

Reply:
Moved to `__experimental` namespace ✅